backend/consensus/worker.py

```
@@ -0,0 +1,734 @@
+                        node_factory,
+                        validators_snapshot,
+                    )
+
+            session.commit()
+            logger.info(
+                f"[Worker {self.worker_id}] Successfully processed transaction {transaction.hash}"
+            )
+
+        except Exception as e:
+            logger.exception(
+                f"[Worker {self.worker_id}] Error processing transaction {transaction_data['hash']}: {e}"
```

Ruff (0.14.0)

Redundant exception object included in `logging.exception` call

(TRY401)